invenio_app_ils/circulation/mail/messages.py

```diff
@@ -16,7 +16,7 @@
 class LoanMessage(BlockTemplatedMessage):
     """Loan message."""
 
-    templates_base_dir = "invenio_app_ils_mail/circulation"
+    templates_base_dir = "invenio_app_ils_assets/circulation"
```

Comment:
any reason to change the name? because if you wanted to generalise it, because we plan to put other assets than just e-mails in there, maybe they should be namespaced here by category f.e. /emails /static etc

Reply:
Thanks @kprzerwa I reverted it because it broke tests, @ntarocco as I see it we have a few options here.

- should we just put everything under `invenio_app_ils_mail` (although we shouldn't)
- should we introduce a new blueprint only for static
- if I am not wrong, all theses configurations templates, static, is part of the invenio-theme. Should we introduce `theme` module?

What do you think?